proto/store/plan_check_run.proto

```diff
@@ -0,0 +1,67 @@
+syntax = "proto3";
+
+package bytebase.store;
+
+option go_package = "generated-go/store";
+
+message PlanCheckRunConfig {
+  int32 sheet_id = 1;
+  int32 database_id = 2;
+}
```

Comment:
what's this for?

Reply:
This stores which sheet and database the plan check run uses.

Reply:
okay IC it's a store layer proto.